### ISSUE TITLE
jsonb format in migration files?

### ISSUE THREAD

**Issue by kaymo4** [SpinaCMS/Spina#681]
I am not sure what is this 'jsonb' format in the migration files, I had to change them to json to run migrate with no error....I am using mysql for DB.

**Comment by Bramjetten** [comment]
Jsonb stands for Json Better. It’s a column type for Postgres. Spina requires Postgres.

**Comment by kaymo4** [comment]
Thanks! I am not too familiar with Postgres but was able to install it.
I am trying to run the install suggested but ran into issues:
active record migration are versioned with different rails, I cloned and edited
then I am struggling with dependencies that are not compatible with ruby 3 I am using:
   spina was resolved to 2.0.2, which depends on
      jsonb_accessor (>= 1.0.0) was resolved to 1.1.0, which depends on
        Ruby (~> 2.0)

Any way to resolve this easily?

Thanks again 
Kay
